src/Footer/Footer.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Button,
+  Col,
+  Row,
+} from '@folio/stripes/components';
+
+import css from './Footer.css';
+
+const Footer = ({ resetDetails, form }) => {
+  return (
+    <div className={css.footer}>
+      <Row>
+        <Col xsOffset={1} xs={3}>
+          <Button
+            id="cancel"
+            type="button"
+            buttonStyle="default mega"
+            onClick={() => {
+              form.change('patronBarcode', '');
+              resetDetails();
+            }}
+          >
+            <FormattedMessage id="ui-reading-room.cancel" />
+          </Button>
+          <Button
+            id="deny-access"
+            type="submit"
+            buttonStyle="danger mega"
+            onClick={() => {}}
```

Comment:
Why do you need useless function here?

Reply:
Hello @artem-blazhko 
I missed to add certain information in the PR description. I have updated it now.

Please note that the above jira is a subtask of a [story ](https://folio-org.atlassian.net/browse/UIRR-4)- hence few lines may seem insignificant. They will be improved in the scope of further sub tasks.